changepointmodel/core/calc/tstat.py

```diff
@@ -19,7 +24,7 @@ def threepc(
     pred_y: OneDimNDArrayField,
     slope: float,
     changepoint: float,
-) -> float:
+) -> SingleSlopeTStat:
     assert changepoint is not None
```

Comment:
We can ditch the assertions here... this was for the runtime in app, like a sanity check...